requirements.txt

```diff
@@ -0,0 +1,55 @@
+antlr4-python3-runtime==4.7.2
+asttokens==2.4.1
+Cartopy==0.23.0
+certifi==2024.6.2
+cf-units==3.2.0
+cftime==1.6.4
+click==8.1.7
+cloudpickle==3.0.0
+contourpy==1.2.1
+coverage==7.6.0
+cycler==0.12.1
+dask==2024.6.0
+decorator==5.1.1
+executing==2.0.1
+f90nml==1.4.4
+fonttools==4.53.0
+fsspec==2024.6.0
+iniconfig==2.0.0
+ipython==8.25.0
+jedi==0.19.1
+Jinja2==3.1.4
+kiwisolver==1.4.5
+locket==1.0.0
+MarkupSafe==2.1.5
+matplotlib==3.9.0
+matplotlib-inline==0.1.7
+mule @ git+https://github.com/metomi/mule@cce4b99c7046217b1ec1192118a786636e0d8e54#subdirectory=mule
```

Comment:
Is this newer than what conda (2020.01.1 on the coecms channel) could provide?

Reply:
To (partially) answer my own question, after commenting out the mule requirement (which `conda` doesn't understand), I was left with the following packages unable to be found:
```
  - xxhash==3.4.1
  - stack-data==0.6.3
  - pure-eval==0.2.2
  - matplotlib==3.9.0
  - antlr4-python3-runtime==4.7.2
```

The channel list for this attempt was:
```
 - accessnri
 - conda-forge
 - http://ssb.stsci.edu/astroconda
 - defaults
 - coecms
```
(noting that some complaint about `coecms` was made).